### PR TITLE
Add How to test locally section to readme

### PR DESCRIPTION
**Pull request by timbussmann**
To document that this now requires a SQL server to run the tests successfully.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,10 @@ MSMQ transport for NServiceBus
 NOTE: As Microsoft is not making MSMQ available for .NET Core, building new systems using MSMQ is not recommended. See https://particular.net/blog/msmq-is-dead
 
 Documentation can be found at <https://docs.particular.net/transports/msmq/>.
+
+## How to test locally
+
+* MSMQ needs to be installed and enabled on the current machine.See the [MSMQ configuration documentation](https://docs.particular.net/transports/msmq/?version=msmqtransport_2#msmq-configuration) on how to install MSMQ.
+* A MSSQL Server needs to be accessible either via:
+  * A local SQL Express installation containing a `nservicebus` database.
```
Comment (timbussmann) [inline README.md:13]: do you also need to create the database manually upfront or will SQL Express create it on the fly?

File: README.md [hunk 2]
```diff
@@ -5,3 +5,10 @@ MSMQ transport for NServiceBus
...
+  * A custom SQL Server connection string in provided via the `SqlServerConnectionString` environment variable
```
Comment (timbussmann) [inline README.md:14]: it looks like this doesn't even seem to work with a remote SQL instance running on a container given the tests seem to rely on MSDTC, is that correct?

File: README.md (hunk 1 above)
Comment (lailabougria) [reply README.md:13]: This should be created upfront, the [tables will be created](https://github.com/Particular/NServiceBus.Transport.Msmq/blob/328ca2c1863ed20cac20b51e8e8d88644d294cf7/src/NServiceBus.Transport.Msmq/DelayedDelivery/Sql/TimeoutTableCreator.cs), but the database is assumed to be available.

File: README.md (hunk 2 above)
Comment (lailabougria) [reply README.md:14]: That's a good point, I hadn't considered that since I have a SQL express running which i am using

**Review by SzymonPobiega — APPROVED** [review]

**Review by lailabougria — APPROVED** [review]

**Pull request merged**